<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaintApplicationClass</class>
 <widget class="QMainWindow" name="PaintApplicationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PaintApplication</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>2</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" rowspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="Circle_Button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>C:/Users/PRAVENDRA RANA/Downloads/Circle.jpg</normaloff>C:/Users/PRAVENDRA RANA/Downloads/Circle.jpg</iconset>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Rectangle_Button">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>C:/Users/PRAVENDRA RANA/Downloads/rectangle-shape-area-clip-art-pictures-of-sandboxes-a252381690ad35aa33792f02dee0c40f.png</normaloff>C:/Users/PRAVENDRA RANA/Downloads/rectangle-shape-area-clip-art-pictures-of-sandboxes-a252381690ad35aa33792f02dee0c40f.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>20</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Line_Button">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>C:/Users/PRAVENDRA RANA/Downloads/636px-Draw-1-black-line.svg.png</normaloff>C:/Users/PRAVENDRA RANA/Downloads/636px-Draw-1-black-line.svg.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>603</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PaintApplication.qrc"/>
 </resources>
 <connections/>
</ui>
